src/sentry/integrations/slack/message_builder/notifications/daily_summary.py

```diff
@@ -39,9 +39,10 @@ def linkify_error_title(self, group):
             params={"referrer": self.notification.get_referrer(ExternalProviders.SLACK)}
         )
         title = build_attachment_title(group)
-        attachment_text = self.get_attachment_text(group).replace("\n", " ")
+        attachment_text = self.get_attachment_text(group)
```

Comment:
🤷 or we could modify the `get_attachment_text` to always return a string?

Reply:
That might have unexpected consequences in other usages of it I don't wanna mess with, though that does remind me that once we GA block kit I want to rename this to `get_block_text` or something.